exec-agent/docs/rest_api.md

```diff
@@ -0,0 +1,231 @@
+REST API
+===
+
+Process API
+---
+
+### Start a new process
+
+#### Request
+
+_POST /process_
+
+- `channel`(optional) - the id of the channel which should be subscribed to the process events
+- `types`(optional) - works only in couple with specified `channel`, defines
+the events which will be sent by the process to the `channel`. Several values may be specified,
+e.g. `channel=channel-1&types=stderr,stdout`. Possible type values:
+    - `stderr` - output from the process stderr
+    - `stdout` - output from the process stdout
+    - `process_status` - the process status events(_started, died_)
+
+
+```json
+{
+    "name" : "build",
+    "commandLine" : "mvn clean install",
+    "type" : "maven"
+}
+```
+
+#### Response
+
+```json
+{
+    "pid": 1,
+    "name": "build",
+    "commandLine": "mvn clean install",
+    "type" : "maven",
+    "alive": true,
+    "nativePid": 9186,
+}
+```
+- `200` if successfully started
+- `400` if incoming data is not valid e.g. name is empty
+- `404` if specified `channel` doesn't exist
+- `500` if any other error occurs
+
+
+### Get a process
+
+#### Request
+
+_GET /process/{pid}_
+
+- `pid` - the id of the process to get
+
+#### Response
+
+```json
+{
+    "pid": 1,
```

Comment:
No, i thought it was obvious from the process.start method, which goes first in the doc and contains both `nativePid` and `pid` at the same time. But i can add a small description to the documentation about response data.